modules/rest-util/src/blaze/middleware/fhir/resource.clj

```diff
@@ -1,5 +1,7 @@
 (ns blaze.middleware.fhir.resource
-  "JSON/XML deserialization middleware."
+  "Resource deserialization middleware.
```

Comment:
please leave it, it's not part of that PR

Reply:
FHIR Resource deserialization middleware.
...